cmd/geth/consolecmd.go

```diff
@@ -43,7 +43,7 @@ var (
 		Description: `
 The Geth console is an interactive shell for the JavaScript runtime environment
 which exposes a node admin interface as well as the Ðapp JavaScript API.
-See https://github.com/ethereum/go-ethereum/wiki/Javascipt-Console.`,
```

Comment:
Please add the missing 'r'

Reply:
Haha, can't believe I missed that!  Let me know if there's anything else I need to do before this can be merged (stop gitcop from complaining?)  This is my first contribution to the project.